### DISCUSSION TITLE
Wanted to know main difference of running Lighthouse audit through dev tools and CLI

### DISCUSSION THREAD

**Discussion by prashanthramanna** [GoogleChrome/lighthouse#15604] [Q&A]
### FAQ

- [X] Yes, my issue is not about [variability](https://github.com/GoogleChrome/lighthouse/blob/main/docs/variability.md) or [throttling](https://github.com/GoogleChrome/lighthouse/blob/main/docs/throttling.md).
- [X] Yes, my issue is not about a specific accessibility audit (file with [axe-core](https://github.com/dequelabs/axe-core) instead).

### URL

https://qatirehubonline.com

### What happened?

just want to know the main difference between running lighthouse through dev tools and through CLI

will the page score or metrics (First contentful paint, Time to interact ) will differ ? 
Could you please provide insights on running lighthouse in different modes, Pros and cons

### What did you expect?

want to run lighthouse through CLI before to that want to understand if there is any major differences or major impact

### What have you tried?

have tried running lighthouse both on dev tools and CLI and have observed the page score varies between the two

### How were you running Lighthouse?

CLI, Chrome DevTools

### Lighthouse Version

11.0

### Chrome Version

119

### Node Version

18.18

### OS

Windows

**Comment by brendankenny** [comment]
By design these modes are intended to run Lighthouse as similarly as possible. The metrics, audit results, etc will largely be the same.

In practice there may be some differences because the CLI has a bit more control over the experience due to being able to launch a new Chrome instance for testing with
- a fresh Chrome profile
- with no additional tabs potentially doing work in the background
- with other background activity minimized as much as possible

There are a few other things that potentially affect performance like data stored in local storage, though the report you get through DevTools will warn you if that existed and potentially affected performance.

> have tried running lighthouse both on dev tools and CLI and have observed the page score varies between the two

You should generally see less variability between two runs of Lighthouse from the CLI than from two runs via DevTools or one from DevTools and one from the CLI. However, variability is inevitable due to how machines, network, and content changes from minute to minute when accessing a particular URL. Some more background here: https://github.com/GoogleChrome/lighthouse/blob/main/docs/variability.md
